eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -17,14 +17,6 @@ steps:
     inputs:
       useGlobalJson: true
       performMultiLevelLookup: true
-  - task: UseDotNet@2 # We need .NET 7 for the .NET 7 tests
```

Comment:
We don't need these now because we're using the .NET 8 SDK as the repository standard?

Reply:
Yeah this was just downloading .NET 7 to be able to run the tests, .NET 8 is used everywhere now.